symbolic-base/symbolic-base.cabal

```diff
@@ -230,11 +232,14 @@ library
       ZkFold.Symbolic.Fold
       ZkFold.Symbolic.Interpreter
       ZkFold.Symbolic.MonadCircuit
-    if arch(javascript)
+    if arch(javascript) || arch(wasm32)
       js-sources:
         js-src/blake2b.js
 
-    if arch(javascript)
+    if arch(wasm32)
+      build-depends:
+        ghc-experimental
+    if arch(javascript) || arch(wasm32)
       build-depends:
```

Comment:
Not a comment on the PR, but a comment on the future to deduplicate the declaration of dependencies in this file